Sources/BottomSheetView.swift

```diff
@@ -385,7 +399,7 @@ public final class BottomSheetView: UIView {
             for: targetOffsets,
             at: currentTargetOffsetIndex,
             in: superview,
-            isDismissible: isDismissable
+            isDismissible: dismissalDelegate != nil
```

Comment:
Still not entirely sure about this argument name. What about `targetMaxY: Bool` or `targetMaxHeight: Bool`. That makes it more clear what it actually does.